app/src/main/res/values/strings.xml

```diff
@@ -692,4 +692,17 @@ Upload your first media by tapping on the add button.</string>
   <string name="achievements_unavailable_beta">Achievements are only available in the prod flavor.</string>
   <string name="leaderboard_unavailable_beta">The leaderboard is only available in the prod flavor.</string>
   <string name="copyright_popup">Please only upload pictures you have taken by yourself. Uploaders of copyrighted images will be blocked. This applies to the beta flavor too. Thank you for testing the app!</string>
+  <string name="select_feedback_data_choice">Please deselect any information that you are not comfortable sharing publicly</string>
+  <string name="api_level">API level</string>
+  <string name="android_version">Android version</string>
+  <string name="device_manufacturer">Device manufacturer</string>
+  <string name="device_model">Device model</string>
+  <string name="device_name">Device name</string>
+  <string name="network_type">Network type</string>
+  <string name="wikimedia_user_name">Wikimedia user name</string>
+  <string name="thanks_feedback">Thanks for giving feedback</string>
+  <string name="error_feedback">Error while sending feedback</string>
+  <string name="enter_description">What is your feedback?</string>
+  <string name="feedback_header">Feedback for version</string>
```

Comment:
This string is not used in the UI, right?
So I think it would be better as a hard-coded string. Getting this string localized would waste the translators' time and unnecessarily grow the APK size a little bit.
Also, would you mind adding a space at the end of the string? (right now it generates titles like `Feedback for version3.1.1` at https://commons.wikimedia.org/wiki/Commons:Mobile_app/Feedback